src/error_impls.rs

```diff
@@ -5,6 +5,7 @@
 // <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
+#![cfg_attr(docsrs, doc(cfg(feature = "std")))]
```

Comment:
This line is not needed.

Reply:
This is also what I thought, but apparently impl support got added.